dags/openshift_nightlies/scripts/install/hypershift.sh

```diff
@@ -122,18 +123,35 @@ postinstall(){
     kubectl create secret generic $HOSTED_KUBEADMIN_NAME --from-literal=KUBEADMIN_PASSWORD=${PASSWORD}
     echo "Wait till Hosted cluster is ready.."
     export KUBECONFIG=./kubeconfig
-    node=0
-    while [ $node -lt $COMPUTE_WORKERS_NUMBER ]
-    do
-        node=$(oc get nodes | grep worker | grep -i ready | wc -l)
-        echo "Available nodes on cluster - $HOSTED_CLUSTER_NAME ...$node"
-        sleep 300
-    done
+    export NODEPOOL_SIZE=$(cat ${json_file} | jq -r .hosted_cluster_nodepool_size)
+    if [ "${NODEPOOL_SIZE}" == "0" ] ; then
+        echo "None type cluster with nodepool size set to 0"
+    else
+        node=0
+        while [ $node -lt $COMPUTE_WORKERS_NUMBER ]
```

Comment:
@mukrishn is there a default timeout in this dag/task or does one need to get added here? Otherwise we have a potential infinite loop if the nodes dont launch.

Reply:
yeah right, possibly it could get in to infinite loop. I will change this with a timeout instead of checking for nodes.